todo-mvc/src/containers/TodoAppContainer.ts

```diff
@@ -0,0 +1,44 @@
+import { Container } from '@dojo/widget-core/Container';
+import { TodoApp } from './../widgets/TodoApp';
+import uuid from '@dojo/core/uuid';
+import { Store } from 'le-store/store';
+import {
+	addTodoProcessWithPost,
+	editTodoProcess,
+	toggleTodoProcess,
+	deleteTodoProcess,
+	toggleAllTodoProcess,
+	clearCompletedProcess,
+	saveTodoProcess,
+	todoInputProcess,
+	clearFailedProcess
+} from './../processes/todoProcesses';
+
+function getProperties(store: Store, properties: any) {
+	const { get, createExecutor } = store;
+
+	return {
+		clearFailed: createExecutor(clearFailedProcess),
+		addTodo: createExecutor(addTodoProcessWithPost, (label: string) => {
```

Comment:
`createExecutor` create a "deferred" executor that when called with arguments will start the process passed, i.e. `addTodoProcessWithPost`. 

The arguments that the deferred executor is called with will be passed to the `transformer` function passed as the second argument to `createExecutor`. The return from the `transformer` will be passed to every command in the process.